test/suite/test_tiered08.py

```diff
@@ -104,6 +104,14 @@ def verify(self, key_count):
         c.close()
 
     def test_tiered08(self):
+
+        # FIXME-WT-9823, WT-9837
```

Comment:
I thought one of these was closed as a dup. If so, remove that ticket from the comment. If not, then I think both tickets need to have the prefix, so `FIXME-WT-9837` in order for the script to detect it. (I'm not entirely sure about that but it seems a possibility.)